conanfile.py

```diff
@@ -274,6 +275,11 @@ def requirements(self):
             for req in self._um_data()["internal_requirements"]:
                 self.requires(req)
 
+    def build_requirements(self):
+        if self.settings_build.os == "Windows" and not self.conf.get("tools.microsoft.bash:path", default=False, check_type=bool):
+            self.tool_requires("msys2/cci.latest")
```

Comment:
Maybe a good idea to only do this if translations are required? (Have an option to be able to build without translations.)